Reformat ivalue_inl.h and ivalue.h

wayi1 opened the pull request:
Summary: Want to separate the real changes in this file from noisy reformatting changes, so check in this reformatting first.

Test Plan: N/A

Differential Revision: D24246841

pritamdamania87 commented:
Looks like there are some build failures, could you look into those? Also, I'm assuming this is just running clang-format on the respective files?

SciPioneer commented:
> Looks like there are some build failures, could you look into those? Also, I'm assuming this is just running clang-format on the respective files?

Yes, only clang-format is applied. Not sure why the first version has failures. Now confirmed no failure by running the previously failed tests.

Commit:
Reformat ivalue_inl.h and ivalue.h (#46174)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/46174

Want to separate the real changes in this file from noisy reformatting changes, so check in this reformatting first.

Test Plan: N/A

Reviewed By: pritamdamania87

Differential Revision: D24246841

fbshipit-source-id: 398f0f25b6ab77a901d59cbc9de614a7b5295173

rohan-varma reviewed (approved):
LGTM on green
